ou_dedetai/wine.py

```diff
@@ -376,6 +376,42 @@ def install_msi(app: App):
     return run_wine_proc(wine_exe, app, exe="msiexec", exe_args=exe_args)
 
 
+def run_winetricks(app: App, *args):
+    cmd = [*args]
+    if "-q" not in args and app.conf.winetricks_binary:
+        cmd.insert(0, "-q")
+    logging.info(f"running \"winetricks {' '.join(cmd)}\"")
+    process = run_wine_proc(app.conf.winetricks_binary, app, exe_args=cmd)
+    if process is None:
+        app.exit("Failed to spawn winetricks")
+    logging.debug(f"Waiting for \"winetricks {' '.join(cmd)}\" To complete")
+    process.wait()
+    if process.returncode != 0:
+        app.exit(f"\"winetricks {' '.join(cmd)}\" Failed!")
+    logging.info(f"\"winetricks {' '.join(cmd)}\" DONE!")
+    logging.debug(f"procs using {app.conf.wine_prefix}:")
+    for proc in utils.get_procs_using_file(app.conf.wine_prefix):
+        logging.debug(f"winetricks proc: {proc=}")
+    else:
+        logging.debug('winetricks proc: <None>')
+
+
+def install_fonts(app: App):
+    """Installs all required fonts:
+    
+    - arial
+    """
```

Comment:
Would it be worth making the font name (or list of names) an argument to the function? That would let this work in a more general way if other fonts are ever needed.

Reply:
There could be a sub function that takes in a list, this function is meant to "just handle it"

We don't want the implementation detail that arial is the only want installed strewn across the code, it makes sense to have it in one place, and that place is here. A sub-function would be fine, I achieved a similar result by using a list a couple lines down. Passing in an argument might be useful to unit tests in the future - but no real point if there is only one.